provider/spark.go

```diff
@@ -37,30 +37,45 @@ type SparkExecutorType string
 
 const (
 	EMR        SparkExecutorType = "EMR"
-	Databricks                   = "DATABRICKS"
+	Databricks SparkExecutorType = "DATABRICKS"
```

Comment:
Dont need to do this, the type is inferred from the previously declared constant